app/models/waste_carriers_engine/order.rb

```diff
@@ -88,7 +90,11 @@ def generate_id
     end
 
     def generate_description
-      self.description = order_items.map(&:description).join(", plus ")
+      self.description = order_items.map(&:description)
+                                    .join(", plus ")
+
+      # Upcase the first letter. (TODO: Use .upcase_first when we switch to Rails 5)
+      description[0] = description[0].capitalize
```

Comment:
Why using `self.` above but not below?

Reply:
Honestly I'm not sure - it looks weird to me as well! Rubocop complains if the second one is `self.description` and says it is redundant, but the first doesn't work without it 🤷‍♀

Reply:
rubocop nonsense 😂 
I was thinking, what if we make one method for the setting and one method for generating the description? I think it will be clearer and rubocop should not complain! So we can have:
```
# Public interface
def set_description
  self.description = generate_description
end


...
# Private interface 
def generate_description
  description = ""
  
  # Do stuff

  description
end
```

Reply:
Ooh, I like this 👍 Will sort it out!

Reply:
Done: https://github.com/DEFRA/waste-carriers-engine/pull/733/commits/39834d7f2ccfae68f4e0ccae2c65b767fe7d19c0